src/main/index.ts

```diff
@@ -239,13 +239,17 @@ function onAppReady(): void {
       ((url.hostname   === 'localhost' || url.hostname   === '127.0.0.1') && // Treat "localhost" and "127.0.0.1" as the same hostname
        (remoteHostname === 'localhost' || remoteHostname === '127.0.0.1')))) {
       callback({
-        ...details.responseHeaders,
-        responseHeaders: 'script-src \'self\'',
+        responseHeaders: {
+          ...details.responseHeaders,
+          'Content-Security-Policy': ["script-src 'self'"]
+        }
       });
     } else {
       callback({
-        ...details.responseHeaders,
-        responseHeaders: 'script-src \'self\'',
+        responseHeaders: {
+          ...details.responseHeaders,
+          'Content-Security-Policy': ["script-src 'self'"]
+        },
         cancel: true
```

Comment:
The type updates revealed that the previous CSP policy was probably broken. Commit 2a528e5fa69d579c1fc6cf61cc30440c4cf45005 ("feat: add "back" process and make it handle the preferences") seems to have worked around the broken CSP by adding a manual check. So I guess the CSP and the manual check now serve the same purpose?

I feel like this is fine for now but we might want to figure out if both are needed together. @TBubba what do you think?

Reply:
Whoops, that was definitely broken.

I'm not really sure how the CSP works or why it was set to the value it was. It probably needs some looking into at some point (because I'm pretty sure Electron keeps asking for it in the console), but I think this looks fine for now.